Subject: [Nouveau] [PATCH] gallium/nv50: use 0x8697 class on NVAF
Upstream: No (but fixed in mesa master, so drop after 7.10)
Addresses: https://bugs.launchpad.net/ubuntu/+source/xserver-xorg-video-nouveau/+bug/723012

Reported-and-tested-by: Alan Pope
---
 src/gallium/drivers/nv50/nv50_reg.h    |    4 ++++
 src/gallium/drivers/nv50/nv50_screen.c |    3 +++
 2 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/src/gallium/drivers/nv50/nv50_reg.h b/src/gallium/drivers/nv50/nv50_reg.h
index 949838b..90d77e5 100644
--- a/src/gallium/drivers/nv50/nv50_reg.h
+++ b/src/gallium/drivers/nv50/nv50_reg.h
@@ -1685,6 +1685,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
 
+#define NVAFTCL										0x00008697
+
+
+
 #define NV50_COMPUTE									0x000050c0
 
 #define  NV50_COMPUTE_NOP								0x00000100
diff --git a/src/gallium/drivers/nv50/nv50_screen.c b/src/gallium/drivers/nv50/nv50_screen.c
index edc3d54..8069509 100644
--- a/src/gallium/drivers/nv50/nv50_screen.c
+++ b/src/gallium/drivers/nv50/nv50_screen.c
@@ -389,6 +389,9 @@ nv50_screen_create(struct pipe_winsys *ws, struct nouveau_device *dev)
 		case 0xac:
 			tesla_class = NVA0TCL;
 			break;
+		case 0xaf:
+			tesla_class = NVAFTCL;
+			break;
 		default:
 			tesla_class = NVA8TCL;
 			break;
-- 
1.7.4.rc3

_______________________________________________
Nouveau mailing list
Nouveau@lists.freedesktop.org
http://lists.freedesktop.org/mailman/listinfo/nouveau

